src/styles/mixins/_control-base.scss

```diff
@@ -0,0 +1,83 @@
+@import "../../styles/settings/colors";
+@import "../../styles/settings/base";
+@import "../../styles/mixins/control-icon";
+@import "../../styles/mixins/visual-hidden";
+@import "../../styles/mixins/control-transition";
+
+@mixin control-base($block, $border-radius: 0) {
+  .#{$block} {
```

Comment:
I would remove the `$block` and use the intrinsic `&` provided by sass.
Like I did it here:
https://github.com/axa-ch/patterns-library/blob/master/src/styles/mixins/_visibility.scss
And utilized here:
https://github.com/axa-ch/patterns-library/blob/master/src/styles/trumps/_visibility.scss